### PR TITLE
Makefile for creating xpis for each platforms.

### PR DESCRIPTION
**Pull request by hiikezoe**
The part of extracting the version number from manifest.json refered to a
post [1] in Stack Overflow.

[1] https://stackoverflow.com/questions/1955505/parsing-json-with-unix-tools#comment42500081_1955505

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+ARCHS=linux linux64 mac64 win32
+MANIFEST=manifest.json
+
+VERSION=`grep -Po '"'"version"'"\s*:\s*"\K([^"]*)' $(MANIFEST)`
```
Comment (juliandescottes) [inline Makefile]: It looks like `-Po` is not available for grep on OSX by default.
Either:
- use something that would work on OX as well. I am very bad with scripts, but following seems to work on my machine:
  `grep -Eo '"version"\:\s*"[^"]+' manifest.json | grep -Eo '\d.*'`
- force OSX users to update grep for instance via
  `brew install grep --with-default-names`
  (and document it accordingly, the best would be to log something if VERSION is empty at this step)

**Review by juliandescottes — CHANGES_REQUESTED** [review]
Thank you so much for continuing the effort on the ADB extension Hiro! Looks like OSX is making our lives difficult again :)

File: Makefile (hunk 1 above)
Comment (hiikezoe) [reply Makefile]: Oh, nice.  the -P option must be a GNU grep specific option, whereas the default grep on MacOSX is BSD one?
I will take the first approach in your suggestions.

Comment (hiikezoe) [reply Makefile]: Hmm \d doesn't work for me.   I will use -Eo '[0-9].*' instead.  I hope it works on MacOSX.

**Comment by juliandescottes** [comment]
Works here as well! Merging.

**Pull request merged**